<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductsWiewWindow</class>
 <widget class="QDialog" name="ProductsWiewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="itemsList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>221</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="legendLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>название/стоимость/есть в наличии</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>113</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>Добавить в корзину</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="addInput"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="messageLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выбирите товар, введите количество и добавьте его в корзину</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>ProductsWiewWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
